City-of-Helsinki/berth-reservations-admin
# VEN-905 | Add delete application to all application types

## Description :sparkles:

* Add delete application to all application types

## Issues :bug:

### Closes :no_good_woman:

* [VEN-905](https://helsinkisolutionoffice.atlassian.net/browse/VEN-905): FE - Add delete Application to all application types

## Testing :alembic:

### Automated tests :gear:️

* Updated relevant tests, added tests for berth applications about delete application button visibility
  * Note that winter storage applications are lacking such tests in general, could be done in a separate scope
* Updated test for changed product service types

### Manual testing :construction_worker_man:

* Delete button should be visible for applications that have not been paid
* Delete button should not be visible for applications that have been paid
